app/scripts/modules/amazon/src/serverGroup/details/serverGroupDetails.aws.controller.js

```diff
@@ -154,6 +154,11 @@ module.exports = angular.module('spinnaker.amazon.serverGroup.details.controller
                 this.changeConfig = {
                   metadata: get(this.serverGroup.entityTags, 'creationMetadata')
                 };
+                if (this.serverGroup.buildInfo.jenkins) {
```

Comment:
TBH, it feels weird to be doing the jenkins check outside of the netflix module, but we're doing it elsewhere so not sure what the best solution here is.  It feels like we should move this into netflix, but not sure how realistic/feasible that is, currently.